templates/dart/lib/client.dart.twig

```diff
@@ -76,17 +59,10 @@ class Client {
 
     Future init() async {
         if(!initialized) {
-          final Directory cookieDir = await _getCookiePath();
-
-          cookieJar = new PersistCookieJar(dir:cookieDir.path);
-
+          cookieJar = new PersistCookieJar(dir:'./cookies');
```

Comment:
I think we don't need the cookie jar for the server integration. No need for any cookie based auth when integrating with the server API, we'll only use API Key & JWT for authentication.

Reply:
@eldadfux Ok